Rakefile

```diff
@@ -5,24 +5,30 @@ require "date"
 
 task default: :formula_and_analytics
 
-desc "Dump formulae data"
+desc "Dump macOS formulae data"
 task :formulae do
   ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1"
   ENV["HOMEBREW_NO_COLOR"] = "1"
   sh "brew", "ruby", "script/generate.rb"
 end
 
+desc "Dump Linux formulae data"
+task :formulae_linux do
+  ENV["HOMEBREW_NO_COLOR"] = "1"
+  sh "brew", "ruby", "script/generate-linux.rb"
+end
+
 desc "Dump cask data"
 task :cask do
   ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1"
   ENV["HOMEBREW_NO_COLOR"] = "1"
   sh "brew", "ruby", "script/generate-cask.rb"
 end
 
-def generate_analytics?
+def generate_analytics?(os = nil)
   return false if ENV["HOMEBREW_NO_ANALYTICS"]
 
-  json_file = "_data/analytics/build-error/30d.json"
+  json_file = "_data/analytics#{"-linux" if os}/build-error/30d.json"
```

Comment:
This naming feels a bit weird. Should either be `#{"-#{os}" if os}` or `#{"-linux" if linux}`

Reply:
I changed how this works: the default `os` value is now `"mac"` and only the linux Rake task sets it to anything else.